.travis.yml

```diff
@@ -1,15 +1,18 @@
 language: python
 python:
-  - "2.6"
+  # - "2.6"
   - "2.7"
   #- "3.1"  # cram does not support Python <3.2.4 anymore
   #- "3.2"
-  #- "3.3"   # but now it does support Python 3.3, according to release note 0.6
+  - "3.3"   # but now it does support Python 3.3, according to release note 0.6
+  - "3.4"
+  - "3.5"
+  - "3.6"
 branches:
   except:
     - piptools-ignore-patch
 install:
-  - "pip install cram --use-mirrors"
-  - "pip install . --use-mirrors"
+  - "pip install cram"
+  - "pip install ."
```

Comment:
This is an OK change. `--use-mirrors` has been removed as of pip 7.0 and it isn't really needed. Would be nice if you could submit this change separately.